src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -102,16 +102,16 @@ func (r *ClusterReconciler) Reconcile(
 		headlessSvc.HeadlessServiceFQDN(),
 		headlessSvc.Key().Name,
 		nodeportSvc.Key(),
-		cert.SecretKey(),
+		pki.NodeCert(),
+		pki.OperatorClientCert(),
 		sa.Key().Name,
 		r.configuratorTag,
 		log)
-	toApply := []resources.Resource{
+	toApply := []resources.Reconciler{
```

Comment:
I think this line should be split into 2 commits. 

1. rename .resource -> reconciler (could be part of prev commit split)
2. introduce pki

Reply:
yeah, you're right, thanks